contrib/integrations/artifactory/plugin-artifactory-release/main.go

```diff
@@ -203,10 +204,13 @@ func (e *artifactoryReleasePlugin) Run(_ context.Context, opts *integrationplugi
 	}, nil
 }
 
-func (e *artifactoryReleasePlugin) createReleaseBundle(distriClient *distribution.DistributionServicesManager, projectKey, workflowName, version, buildInfo string, artifactList, releaseNote string, artifactPromoted []promotedArtifact, artifactoryURL, releaseToken string) (string, string, error) {
+func (e *artifactoryReleasePlugin) createReleaseBundle(distriClient *distribution.DistributionServicesManager, projectKey, workflowName, version, buildInfo string, artifactList, releaseNameSuffix, releaseNote string, artifactPromoted []promotedArtifact, artifactoryURL, releaseToken string) (string, string, error) {
 	buildInfoName := fmt.Sprintf("%s/%s/%s", buildInfo, projectKey, workflowName)
 
 	params := services.NewCreateReleaseBundleParams(strings.Replace(buildInfoName, "/", "-", -1), version)
+	if releaseNameSuffix != "" {
+		params.Name += releaseNameSuffix
```

Comment:
Shoudln't you add a separator and/or sanitize the suffix ? (if there is a / in it for instance).

Reply:
I should but it's something temporary and during the week i will replace this for 2 new parameters (source maturity and target maturity). We can live like that, you will be careful ;)